test/testsuite/Main.hs

```diff
@@ -1,87 +1,24 @@
-{-# LANGUAGE NoImplicitPrelude #-}
-
--- {-# LANGUAGE QuasiQuotes #-}
-
 module Main (main) where
 
-import RIO
-
-import Test.Tasty                   (TestTree, defaultMainWithIngredients, testGroup)
-import Test.Tasty.HUnit             (Assertion, testCase, (@?=))
-import Test.Tasty.Ingredients.Rerun (rerunningTests)
-import Test.Tasty.Runners           (consoleTestReporter, listingTests)
-import Test.Tasty.SmallCheck        (testProperty)
-
--- import Test.Hspec
--- import Test.Hspec.Wai
--- import Test.Hspec.Wai.JSON
+-- Hspec
 
--- import Network.Wai.Handler.Warp (run)
+import qualified Test.Hspec as Hspec
```

Comment:
Why not `Tasty.Hspec`?

Reply:
https://documentup.com/feuerbach/tasty

> It lets you combine your unit tests, golden tests, QuickCheck/SmallCheck
properties, and any other types of tests into a single test suite.

Reply:
Ah right forgot about that. I started with following the Servant guide on testing and went from there. Eventually we removed the Servant stuff, so I was left with this `Test.Hspec`, forgot all about that `tasty` library.

Reply:
Will replace with Tasty tomorrow.